src/bitset.hpp

```diff
@@ -78,6 +87,8 @@ class Bitset {
   size_t Count() const;
   std::string ToIndexSetString() const;
 
+  // Ensures that subsplit is properly ordered.
```

Comment:
Do you mean that it checks? It fails if not?

Reply:
No, it reorders the subsplits in place if they are out of order.  It is used by the subsplit-style constructors in case arguments are given out of order.

Reply:
Aha, I bet you could turn your clear description into a clear method name.

Reply:
Rather than overloading the Bitset() constructor, I made a static Subsplit() constructor, so inplace reordering function is no longer necessary.  Optimized out.